templates/page.about.liquid

```diff
@@ -0,0 +1,55 @@
+{% paginate blogs['about'].articles by 4 %}
+<section id="about-blog">
+  <div class="container">
+    <div class="grid-row">
+      <div id="blog-left" class="grid-item item-s-12 item-m-6">
+        <div class="grid-column">
+          {% for article in blogs['about'].articles %}
+            {% assign mod = forloop.index | modulo: 2 %}
+            {% if mod != 0 %}
+              <div class="blog-item">
+                {{ article.content }}
+              </div>
+            {% else %}
+              <div class="blog-item blog-item-mobile">
+                {{ article.content }}
+              </div>
+            {% endif %}
+          {% endfor %}
+        </div>
+      </div>
+      <div id="blog-right" class="grid-item item-m-6">
+        <div class="grid-column">
+          {% for article in blogs['about'].articles %}
```

Comment:
I dont think this is a good solution. Better to just use masonry or some js than do this double split markup. Is there a reason it is like this?